x-pack/plugins/apm/public/components/shared/Links/apm/ServiceMapLink.tsx

```diff
@@ -11,7 +11,7 @@ export function useServiceMapHref(serviceName?: string) {
   const pathFor = serviceName
     ? `/services/${serviceName}/service-map`
     : '/service-map';
-  return useAPMHref(pathFor);
+  return useAPMHref({ path: pathFor });
```

Comment:
Can you rename `pathFor` to `path` or inline it?
```suggestion
  return useAPMHref({ path });
```